openapi.yml

```diff
@@ -162,6 +162,8 @@ components:
           required:
           - switch
           properties:
+            timeout:
```

Comment:
We should also set the minimum value at least to avoid validation issues or misunderstandings/surprises. 

Good thing that `asyncio.sleep(value)` doesn't crash with negative values unlike `time.sleep(value)`.